gatsby-node.js

```diff
@@ -111,6 +111,8 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
       template = `tutorial`
     } else if (slug.includes(`/docs/`)) {
       template = `docs`
+    } else if (slug.includes(`/eth2/`)) {
```

Comment:
We don't actually want to apply this blanket rule for Eth2 pages. Some of theme should use the static.js template. That's why I added the frontmatter logic above (to check for the `template` field, which is set to `eth2` on the desired pages.